File: atcoder/abc106/b.py
n = int(input())

def solve(num):
    c = 0
    for i in range(1, num + 1):
        if num % i == 0:
            c += 1
    if c == 8 and num % 2 == 1:
        return True
    return False

count = 0
for i in range(1, n + 1):
    if solve(i):
        count += 1

print(count)

File: atcoder/abc106/b.jl
N = parse(Int, readline())

function solve()
	ans = 0
	for i in 1:N
		count = 0
		if mod(i, 2) != 1
			continue
		end
			for j in 1:i
				if mod(i, j) == 0
					count += 1
				end
			end
			if count == 8 
				ans += 1
			end
		end
	println(ans)
end

solve()

